Makefile

```diff
@@ -9,19 +9,16 @@ RELEASE = dev-t4c-manager
 NAMESPACE = t4c-manager
 SESSION_NAMESPACE = t4c-sessions
 PORT ?= 8080
-
+CAPELLA_VERSIONS = 5.0.0 5.2.0 6.0.0
+T4C_CLIENT_VERSIONS = 5.2.0 6.0.0
```

Comment:
```suggestion
# List of Capella versions, e.g.: `5.0.0 5.2.0 6.0.0`
CAPELLA_VERSIONS = 5.2.0
# List of T4C versions, e.g., `5.2.0 6.0.0`
T4C_CLIENT_VERSIONS = 5.2.0
```

The default behavior should not change. The whole building process takes a long time and it's high hurdle for new users.

Reply:
> The default behavior should not change. The whole building process takes a long time and it's high hurdle for new users.

It's either that or they get only one working Capella version and the other versions are just dummies, which also does not give a good impression.

Reply:
But waiting more than one hour for the first deployment is also not really great. Also something for the README to describe that we build only one version, but it can be changed with the environment variable.